components/DetailsDisplay/StyledDetailsDisplay.js

```diff
@@ -0,0 +1,16 @@
+import styled from "styled-components";
+
+const Details = styled.div`
+  width: 100%;
+  display: flex;
+  justify-content: space-around;
+`;
+const Info = styled.p`
+  font-size: 0.9rem;
+  margin: 0.8rem 0;
+  padding: 0 0.5rem;
+  border-radius: 10px;
+  box-shadow: 0px 0px 3px 1px;
```

Comment:
```suggestion
  box-shadow: 0 0 3px 1px;
```
Is that a prettier thing? 🤔